fixed bug with login after email verification

Anasauce opened the pull request.

jrob8577 commented on auth/passport.js:
```diff
@@ -30,6 +30,7 @@ const findUser = ( email, password ) => {
 const strategy = new LocalStrategy( OPTIONS, ( email, password, done ) => {
   findUser( email, password)
     .then( user => {
+      console.log("---->", user)
```
Can you please check out this file and re-push? (Checkout should remove this line and put it in a state that will not register as changed.)

Anasauce commented:
Ok I think I fixed it...maybe not in the preferred order 😁   
1) removed the console.log
2) squashed two commit messages into one and re-pushed

will that work? I wasn't quite sure how to checkout changes after pushing
